src/serialization/set.h

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2017, The Monero Project
```

Comment:
Sorry to poke around, but you commented in https://github.com/monero-project/monero/pull/2134/files#r152916607:
> This one copies substantial code from vector, so it is really 2014-2017

Reply:
Yes, I made the requested change without really thinking at the time. Thanks.